app/services/Countries.scala

```diff
@@ -56,5 +56,5 @@ object Countries {
 
   private def countryCode: String => String = cc => cc.split(":")(1).trim
 
-  val allCountries: List[Country] = countries.filter(c => mdgCountryCodes contains c.countryCode)
+  val allCountries: List[Country] = countries
```

Comment:
Why this change?

Reply:
we do not have requirement to filter the countries and this is one of the reason some of the countries are not allowed.